### DISCUSSION TITLE
flat file records to xml conversion

### DISCUSSION THREAD

**Discussion by sanjay86alld** [nextgenhealthcare/connect#5649] [Q&A]
can we transform records from a sequential file to xml format using mirth ?
I need few examples to start with . I have the XML Schema Definition with me.

**Comment by pacmano1** [comment]
Yes.  And you should find examples using Google.  T

**Reply by sanjay86alld** [reply]
can i get an example or reference to start with ?

**Reply by pacmano1** [reply]
Can you please google and find the dozens of references of how to transform data from one format to another, give it try and post questions you have here?  

You can find many references on the forums and elsewhere for CSV -> JSON, CSV -> HL7, and others.  Your needs aren't that different.

**Reply by pacmano1** [reply]
While this isn't the mirth forums, much of this article applies here on the mirth discussion board: https://forums.mirthproject.io/articles/177082-forum-etiquette\

https://www.youtube.com/watch?v=oIK3Up8Scmg&list=PLF1s9Ns6Z3kXWR33Meaa8YEqzvaY2rd1V is a good example video.  Where she has hl7 in the bottom right hand side of the screen, you will instead select XML as the data type, put the sample XML blob in there and drag drop like she shows.   If you need to create new segments, e.g. for repeating elements, take a look over on the forums, there are many examples.
